dds_cli/__main__.py

```diff
@@ -648,7 +648,7 @@ def list_projects(ctx, username, json, sort, usage):
     + dds_cli.utils.multiple_help_text(item="project owner"),
 )
 @click.option(
-    "--is_sensitive",
+    "--non-sensitive",
     required=False,
     is_flag=True,
     help="Indicate if the Project includes sensitive data.",
```

Comment:
I would probably rephrase the help message slightly as well.

Reply:
"Indicate whether the project contains only non-sensitive data"

Reply:
Will fix 👍🏻

Reply:
Done